wordpress-mobile/WordPress-iOS
# [Stats Refresh] Add Period > Posts and Pages card

Fixes #10875 

This adds the Period Posts and Pages card.
- Displays top 6 results.
- If there are more than 6, `View more` is added.
- Each row contains:
  - Icon
  - Data bar
  - Disclosure icon
- Selecting different filters (DWMY) updates the data.
- If there is no data, `No data for this period` is displayed.

NOTES:
- `View more` will show a temporary alert.
- Selecting a row will show a temporary alert.
- Since the backend does not distinguish between Posts or Pages, for now every row will use the Post icon. (cc @SylvesterWilmott ) (related issue: https://github.com/wordpress-mobile/WordPress-iOS/issues/10886)
- @jklausa  - I added a `SiteStatsPeriodDelegate`. I thought I was going to need it for this card, but I was  mistaken. However, I will need it for other cards, so went ahead and left it. Sorry for the bit of extra code to review, but it'll be worth it! 😄 

To test:

---
On a site with Posts and/or Pages, go to Stats > Days, Weeks, Months, or Years.
- en.blog and Hogwarts are good ones.
- Verify the view is:
<img width="350" alt="pandp" src="https://user-images.githubusercontent.com/1816888/51641304-0a61aa80-1f23-11e9-9c68-334f7975a764.png">

---
On a site with _no_ Posts or Pages, go to Stats > Days, Weeks, Months, or Years.
- Verify the view is:
<img width="350" alt="nopandp" src="https://user-images.githubusercontent.com/1816888/51641405-472da180-1f23-11e9-8ebe-5f8841159a54.png">
